examples/jsm/controls/OrbitControls.js

```diff
@@ -470,9 +472,7 @@ class OrbitControls extends EventDispatcher {
 		const pointerPositions = {};
 
 		function getAutoRotationAngle() {
-
-			return 2 * Math.PI / 60 / 60 * scope.autoRotateSpeed;
-
+			return (2 * Math.PI / 60 * scope.autoRotateSpeed) * _clock.getDelta();
```

Comment:
The problem with using `THREE.Clock` is that when users switch to another tab and then come back to the application after a while, you end up with a huge time delta value that leads to a visible jump. Compare the PRs version with prod to see this effect:

https://raw.githack.com/Issung/three.js/%2326471-orbitcontrols-framerate-independent/examples/webgl2_volume_instancing.html
https://threejs.org/examples/webgl2_volume_instancing

The controls now also produce large time delta values when users enable - disable - enable auto rotation.